modules/ocf_printhost/files/ocfps

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Use the user command pdftops, not the CUPS filter.
+PDFTOPS=/usr/bin/pdftops
+
+if [ "$#" -ne 5 ] && [ "$#" -ne 6 ]; then
+    echo "Usage: $0 job-id user title copies options [file]"
+    exit
+fi
+
+if [ "$#" -eq 5 ]; then
+    $PDFTOPS - - | pstops "$@"
+else
+    $PDFTOPS "$6" - | pstops "${@:1:5}"
+fi
```

Comment:
Oops, I think I deleted part of my response. I originally said that the stuff in my explanation should definitely be in a comment in the script, but that much of it, particularly the generic stuff, should still be in documentation because it's mostly basic info that anyone working with CUPS should know.